app/assets/stylesheets/pages/_categories.scss

```diff
@@ -0,0 +1,283 @@
+.container-main-category {
+  align-items: center;
+  box-sizing: content-box;
+  height: 100%;
+  justify-content: center;
+  overflow-y: scroll;
+  padding-right: 17px;
+  text-align: center;
+  width: 100%;
+  
+  .destroy-show-category {
+    margin-top: 3%;
+    
+    button {
+      background-color: $orange;
+      border-radius: 15px;
+      border: none;
+      color: $black;
+      font-size: 25px;
+      height: 3rem;
+      padding: 1.5%;
+      width: 52%;
+    }
+  }
+  .box-main-category {
+    background: $black-gray;
+    border-radius: 22px;
+    margin-bottom: 5%;
+    margin-left: 30%;
+    margin-right: 30%;
+    margin-top: 5%;
+    padding: 2rem;
+    
+    .title-main-category {
+      color: $white;
+      font-size: 25px;
+      margin-bottom: 5%;
+    }
+    .list-main-category {
+      font-size: 20px;
+      
+      p {
+        font-size: 25px;
+        margin-bottom: 2%;
+      }
+      div {
+        color: $white;
+        margin-bottom: 1%;
+        margin-left: 10%;
+        margin-right: 10%;
+        margin-top: 5%;
+      }
+      a {
+        background-color: $blue;
+        border-radius: 15px;
+        color: $black;
+        font-size: 25px;
+        padding-bottom: 1%;
+        padding-left: 17%;
+        padding-right: 17%;
+        padding-top: 1%;
+        text-decoration: none;
+      }
+    }
+  }
+  .link-main-category {
+    margin-top: 10%;
+    
+    a {
+      background-color: $orange;
+      border-radius: 15px;
+      color: $black;
+      font-size: 30px;
+      padding-left: 5%;
+      padding-right: 5%;
+      text-decoration: none;
+    }
+  }
+}
+
+.container-show-category {
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  
+  .box-show-category {
+    background: $black-gray;
+    border-radius: 22px;
+    margin-left: 30%;
+    margin-right: 30%;
+    margin-top: 5%;
+    padding: 2rem;
+    position: relative;
+    
+    p {
+      color: $white;
+      font-size: 50px;
+    }
+    .opc-show-category{
+      font-size: 30px;
```

Comment:
It will be cool to have global variables with the font size specific to be more responsive, can create a variable like `$font-lg: 30px` to rewrite this line how `font-size: $font-lg` to be more specific about what font-size is used.